lib/orm2-adapter.js

```diff
@@ -87,35 +92,35 @@ export default class NodeORM2Adapter extends ORMAdapter {
   }
 
   saveRecord(model) {
-    return fromNode(model.record.save.bind(model.record));
+    return fromNode((cb) => model.record.save(cb));
   }
 
   deleteRecord(model) {
-    return fromNode(model.record.remove.bind(model.record));
+    return fromNode((cb) => model.record.remove(cb));
   }
 
   defineModels(models) {
-    // Don't define abstract base classes
-    models = models.filter((Model) => !(Model.hasOwnProperty('abstract') && Model.abstract));
```

Comment:
It's done instead in the `define-orm-models` initializer in Denali itself: https://github.com/denali-js/denali/blob/master/config/initializers/define-orm-models.ts#L21